### PR TITLE
Test create user

### PR DESCRIPTION
**Pull request by Palusha**

### CODE REVIEW

**Review by artempenteskul — APPROVED** [review]

File: admin/tests/test_unit/test_forms/test_login_form.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+from admin.forms import LoginForm
+
+
+def test_login_form(client):
+    form = LoginForm(data={'admin': 'testing', 'password': 'testtest'})
+    assert form.validate()
+
+
+def test_login_form_with_invalid_short_data(client):
```
Comment (ivan-dubei) [inline admin/tests/test_unit/test_forms/test_login_form.py]: Split this test into two independent cases.

File: admin/tests/test_unit/test_models/test_author_model.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    with pytest.raises(ValidationError) as exception:
+        author = Author(name='immortal', birthdate='2000', death_date='1995')
+        author.save()
+        author.delete()
+    assert 'Death date should be greater then birthdate' in exception.value.message
+
+
+def test_get_not_existing_author(client):
+    author = Author.objects(name='some not existing author')
+    assert isinstance(author, BaseQuerySet)
```
Comment (ivan-dubei) [inline admin/tests/test_unit/test_models/test_author_model.py:37]: Recheck if we really use BaseQuerySet in this case

File: admin/tests/test_unit/test_models/test_author_model.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+def test_get_author_by_not_existing_attribute(client):
+    author = Author.objects(gander='male')
+    assert isinstance(author, BaseQuerySet)
```
Comment (ivan-dubei) [inline admin/tests/test_unit/test_models/test_author_model.py:43]: Recheck if we really use BaseQuerySet in this case

File: admin/tests/test_unit/test_routes/test_book_storage.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    app.authorize(LOGIN, PASSWORD)
+    response = app.get_book_storage()
+
+    assert HTTPStatus.OK == response.status_code
+    assert b'Title' in response.data
+    assert b'Author' in response.data
+    assert b'Showing page 0' in response.data
+
+
+def test_get_second_page_in_book_storage(app):
```
Comment (ivan-dubei) [inline admin/tests/test_unit/test_routes/test_book_storage.py]: Change test name. We check pagination it this case.

**Review by ivan-dubei — APPROVED** [review]

**Pull request merged**